commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/frontend/CommerceAccountClayTable.java

```diff
@@ -107,22 +104,57 @@
 				LanguageUtil.get(httpServletRequest, "view"), false, false);
 
 			clayTableActions.add(clayTableViewAction);
+
+			CommerceContext commerceContext =
+				(CommerceContext)httpServletRequest.getAttribute(
+					CommerceWebKeys.COMMERCE_CONTEXT);
+
+			CommerceAccount currentCommerceAccount =
+				commerceContext.getCommerceAccount();
+
+			if (account.getAccountId() !=
+					currentCommerceAccount.getCommerceAccountId()) {
+
+				StringBundler sb = new StringBundler(7);
+
+				sb.append("javascript:setCurrentAccount");
+				sb.append(StringPool.OPEN_PARENTHESIS);
+				sb.append(StringPool.APOSTROPHE);
```

Comment:
These strings should really be combined. No need to separate out the `StringPool` characters when they're preceded by a string literal.